tests/integration/tables/shellbags.cpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <osquery/tests/integration/tables/helper.h>
+
+namespace osquery {
+namespace table_tests {
+class ShellbagsTest : public testing::Test {
+ protected:
+  void SetUp() override {
+    setUpEnvironment();
+  }
+};
+
+TEST_F(ShellbagsTest, test_sanity) {
+  // QueryData const rows = execute_query("select * from shellbags");
+  // QueryData const specific_query_rows =
+  //    execute_query("select * from shellbags where path like '%This PC%'");
+
+  // ASSERT_GT(rows.size(), 0ul);
+  // ASSERT_GT(specific_query_rows.size(), 0ul);
```

Comment:
Can we uncomment some of this test?

Reply:
i can uncomment it, but since shellbags are only created when using Windows Explorer the test will likely fail  
In my initial PR i enabled them, but the tests failed at the shellbag tests

Reply:
Ah yes, that makes sense. How about if the test always runs the `execute_query` and then only makes the other assertions if the result is nonempty?

Reply:
i added a check to validate the query if the results is nonempty